app/api/connectors/httpparsers/HttpParser.scala

```diff
@@ -58,7 +58,7 @@ trait HttpParser extends Logging {
   def parseErrors(response: HttpResponse): DownstreamError = {
     val singleError         = response.validateJson[DownstreamErrorCode].map(err => DownstreamErrors(List(err)))
     lazy val multipleErrors = response.validateJson(multipleErrorReads).map(errs => DownstreamErrors(errs))
-    lazy val bvrErrors      = response.validateJson(bvrErrorReads).map(errs => OutboundError(BVRError, Some(errs.map(_.toMtd))))
+    lazy val bvrErrors      = response.validateJson(bvrErrorReads).map(errs => OutboundError(BVRError, Some(errs.map(_.toMtd(response.status)))))
```

Comment:
The purpose of the `OutboundError` is to effectively bypass error mapping in the service layer i.e. to force sending out a specific error. Given this, using the downstream response status here might mislead readers into thinking that it is the status code to be used (otherwise why set it?).